### ISSUE TITLE
inter loss

### ISSUE THREAD

**Issue by bayminbx** [elvisyjlin/RelGAN-PyTorch#7]
I want to only the attribute "Smiling". Do you have some advice?The result of relgan(keras version) is poor...
